packages/itwin/measure-tools/vite.config.ts

```diff
@@ -0,0 +1,30 @@
+import { defineConfig } from "vite";
+
+import react from "@vitejs/plugin-react";
+
+export default defineConfig({
+  plugins: [react()],
+  // look for all static in the dist folder
+  publicDir: "./public",
+  assetsInclude: "./public/*",
+  test: {
+    globals: true,
+    environment: "jsdom",
+    setupFiles: ["src/test/measure-tools/_Setup.test.ts"],
+    include: ["src/test/measure-tools/**/*.test.*"],
+    exclude: ["src/test/measure-tools/_Setup.test.ts"],
+    coverage: {
+      exclude: ["lib/*", "src/test/*"],
+      include: ["src/*"],
+      provider: "istanbul",
+      reportsDirectory: "lib/coverage",
+      reporter: ["html", "text-summary"],
+      thresholds: {
+        statements: 40,
+        branches: 30,
+        functions: 40,
+        lines: 40,
```

Comment:
why the increased values in branches and lines? package.json previously only had 20 for all values

Reply:
This is not a bad thing, honestly the closer we get to complete coverage the better

Reply:
Yeah, I adjusted the thresholds to better reflect the current state of the package. That'll force subsequent contributions to at least think of the coverage.